lldb/test/API/tools/lldb-dap/instruction-breakpoint/Makefile

```diff
@@ -0,0 +1,6 @@
+CXX_SOURCES := main-copy.cpp
+CXXFLAGS_EXTRAS := -O1 -g
```

Comment:
why don't you use -O0?

Reply:
I just verified with -O1, so same has been submitted.
I have changed it to "-O0"